templates/deb/postinst_upgrade.sh.erb

```diff
@@ -8,18 +8,26 @@ after_upgrade() {
 <%=  script(:after_upgrade) %>
 <% end -%>
 
-<% if attributes[:deb_systemd] -%>
+<%# if any systemd services specified, loop through and start them -%>
+<% unless attributes[:deb_systemd].empty? -%>
```

Comment:
For future style note, I prefer to avoid this kind of usage where a double-negative is used. I find it hard to parse (in my head). By double-negative I mean `unless` being a negative of `if` and `.empty?` being the negative of `.any?`. No change required at this time for this line.

"unless empty" can be written without negatives as "if any": `if attributes[:deb_systemd].any?`

In general, I prefer to avoid `unless` in most places with exception to an inline use `foo(..) unless condition` because I find `unless` a bit of a mental twist to read.

Reply:
I've changed this to suit your style :)